### ISSUE TITLE
Release notes without an image tag cause `NullReferenceException` when retrieving GPU information

### ISSUE THREAD

**Issue by clovdystackflow** [ElPumpo/TinyNvidiaUpdateChecker#151]
first time using this, when i open the exe cmd shows for a split second then is gone.

**Comment by clovdystackflow** [comment]
fixed.. this happens now though
![imagen](https://user-images.githubusercontent.com/100868740/213931908-05fc260d-1e01-4064-a3e8-ac452002633f.png)

**Comment by ElPumpo** [comment]
Hi you need the [.NET Desktop Runtime 7 x86](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)

**Comment by clovdystackflow** [comment]
yes, that is indeed everything i get, even with debug 

![imagen](https://user-images.githubusercontent.com/100868740/213932861-688bcb33-debd-4167-a4e6-3efc2d44db3b.png)

**Comment by ElPumpo** [comment]
Huh this is super weird. Can you show your GPU in device Manager?

**Comment by Maverik** [comment]
I'm on Windows 11 - Quadro RTX 5000 and I just ran into this issue as well. I've run the git source and it seems `//img` XPATH query is the the bit throwing NRE.

###Exception:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at TinyNvidiaUpdateChecker.MainConsole.Main(String[] args) in C:\git\TinyNvidiaUpdateChecker\TinyNvidiaUpdateChecker\MainConsole.cs:line 171
```

Nvidia Release notes (`releaseDesc` content @ [MainConsole.cs#L167](https://github.com/ElPumpo/TinyNvidiaUpdateChecker/blob/633a9562ef5421d315f1c5ee469e63ca2806e904/TinyNvidiaUpdateChecker/MainConsole.cs#L167):
```html
<b>NVIDIA RTX Enterprise Production Branch Driver</b> <br />
<br />
Release 525 is a Production Branch release of the NVIDIA RTX Enterprise Driver. This driver provides improvements over the previous branch in the areas of application performance, API interoperability (e.g., OpenCL/Vulkan), and feature support.<br />
<br />
Production Branch drivers are designed and tested to provide long-term stability and availability.  These drivers are ideal for enterprise customers and professional users who require application and hardware certification and regular driver updates for the latest in driver enhancements and security improvements.<br />
<br />
In addition to professional workstation features, this version of the Production Branch driver contains all the benefits of the NVIDIA Studio Driver of the same version.<br />
<br />
<b>New Features in Version R525 U4</b> <br />
<ul>
  <li>Adds the latest performance improvements, bug fixes, and driver enhancements </li>
  
</ul>
<br />
<b>Fixed Issues in Version 525 U4</b> <br />
<ul>
  <li>[Adobe][Photoshop]: application freezes after driver upgrade to version 528.02 </li>
<li>[Adobe][Premiere Pro]: ProRes RAW files are displaying a black preview </li>
<li>[Autodesk][Alias]: Expected transparent window appears opaque </li>
<li>[OTOY][Octane Render]: OctaneBench 2020 fails intermittently due to render engine failure </li>
  
</ul>
<br />
<br />
If you would like to be notified of upcoming drivers for Windows, please <a href="http://www.nvidia.com/object/quadro-driver-notification.html">subscribe here </a>.
```

Hope this helps, feel free to ask for any more specifics that might be helpful.

**Comment by ElPumpo** [comment]
Thanks for the help @Maverik I did not know it would throw an exception if not found.

I am unable to release a update as I work in another country. So you'll have to wait to thursday..

**Comment by Maverik** [comment]
No rush from my end. I've just gone ahead and updated it manually myself. Just wanted to add the detail so you have proper context of whats going wrong.

**Comment by ElPumpo** [comment]
👍 submit a pr for it if youre up for it. Just add a try catch

**Comment by ElPumpo** [comment]
Fixed in v1.16.2. Thanks for the help everyone
